radixdlt-java-common/src/main/java/com/radixdlt/crypto/RadixKeyStore.java

```diff
@@ -81,7 +81,7 @@
  * and clients should make an effort to specify passwords in a secure way.
  */
 @SecurityCritical(SecurityKind.KEY_STORE)
-public final class RadixKeyStore implements Closeable {
+public final class  RadixKeyStore implements Closeable {
```

Comment:
Not sure if this change worth including into the PR

Reply:
Not sure where this is coming from. I havent made this  modification
![image](https://user-images.githubusercontent.com/60398299/111492823-04625f80-8763-11eb-979c-2e057ecf5674.png)